src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstier/integration/bdd/BddSteps.kt

```diff
@@ -81,7 +81,7 @@ class BddSteps : En {
 
       setupOauth()
       crn = UUID.randomUUID().toString().replace("-", "").substring(0, 7)
-      assessmentId = UUID.randomUUID().toString().replace("\\D+".toRegex(), "").substring(0, 11)
+      assessmentId = UUID.randomUUID().toString().replace("\\D+".toRegex(), "").padEnd(11, 'A').substring(0, 11)
```

Comment:
I had an intermittent failure where we couldn't substring after taking the digits out